README.hbs

```diff
@@ -1,10 +1,47 @@
 # node-sass-extra
 
-A drop-in replacement for [node-sass](https://github.com/sass/node-sass) that adds support for globs, promises and more.
-
 [![Commitizen friendly](https://img.shields.io/badge/commitizen-friendly-brightgreen.svg)](http://commitizen.github.io/cz-cli/)
+[![Open Source Love](https://badges.frapsoft.com/os/mit/mit.svg?v=102)](https://github.com/ellerbrock/open-source-badge/)
+[![Open Source Love](https://badges.frapsoft.com/os/v2/open-source.svg?v=102)](https://github.com/ellerbrock/open-source-badge/)
+[![GitHub issues](https://img.shields.io/github/issues/marksmccann/node-sass-extra)](https://github.com/marksmccann/node-sass-extra/issues)
+
+A drop-in replacement for [node-sass](https://github.com/sass/node-sass)' Node API that adds support for globs, promises and more.
+
+:heavy_check_mark: Glob support  
+:heavy_check_mark: Promise support  
+:heavy_check_mark: Write files  
+:heavy_check_mark: Write source maps  
+:heavy_check_mark: Output to directory  
+:heavy_check_mark: Dynamically define output destination  
+:heavy_check_mark: Compile multiple files at once  
+:heavy_check_mark: Compile multiple files into single output  
+:heavy_check_mark: Sync and async support  
+:heavy_check_mark: Non-breaking `node-sass` API  
```

Comment:
@phillipluther Am I missing anything? Can any of the features be written better?

Reply:
This sounds good; I was initially trying to shoehorn the word "concatenation" into "Compile multiple files into single output," but I think that's pretty clear to anybody familiar with `node-sass` limitations.

And anybody unfamiliar probably isn't going to be interested in this.